src/Base/BaseClassPyImp.cpp

```diff
@@ -36,27 +36,32 @@ std::string BaseClassPy::representation() const
 }
 
 
-PyObject*  BaseClassPy::isDerivedFrom(PyObject *args)
+PyObject* BaseClassPy::isDerivedFrom(PyObject* args)
 {
-    char *name{};
-    if (!PyArg_ParseTuple(args, "s", &name))
+    char* name {};
+    if (!PyArg_ParseTuple(args, "s", &name)) {
         return nullptr;
+    }
 
+    // clang-format off
     Base::Type type = Base::Type::fromName(name);
     bool valid = (type != Base::Type::badType() && getBaseClassPtr()->getTypeId().isDerivedFrom(type));
     return PyBool_FromLong(valid ? 1 : 0);
+    // clang-format on
 }

```

Comment:
Why did you choose to deactivate clang-format for this block? It is mere curiosity and to learn something.

Reply:
Sometimes the auto-formatting reduces readability.

Reply:
Because the line is a bit too long clang-format will break it which IMO doesn't look so nice. Now that the class BaseClass has the method isDerivedFrom() the line can be make shorter which makes the formatting off/on superfluous. I will push force a commit.

Reply:
The modified code doesn't require the disabled formatting any more